### ISSUE TITLE
Fix name field in Sprite import panel

### ISSUE THREAD

**Issue by nightm4re94** [gurkenlabs/litiengine#778]
The name panel in the Sprite import panel doesn't show the current spritesheet name or set it when changed.

**Comment by iamllcoolray** [comment]
Hey,

I am interested in resolving this issue. Would y'all be able to provide additional information and/or anything else that could help me?

**Comment by iamllcoolray** [comment]
Hi again,

I just wanted to comment again about my interest in tackling this issue. Can y'all provide additional information and/or anything else that could help me? For example, how I can replicate the bug, what the proper functionality should look like, where in the utiLITI project should start looking in...etc.

I know how busy everyone is so please get back to me when it is most convenient for y'all.

**Comment by nightm4re94** [comment]
Hi, I'm sorry I was not yet able to respond. Having a busy week at work and some fun private events this week. I can't promise to be able to provide further details this week unfortunately.

**Comment by nightm4re94** [comment]
Thanks for your patience.
The problem lies in the [SpritesheetImportPanel](https://github.com/gurkenlabs/litiengine/blob/main/utiliti/src/main/java/de/gurkenlabs/utiliti/swing/dialogs/SpritesheetImportPanel.java#L51):


https://github.com/gurkenlabs/litiengine/assets/26114385/79fdbf83-9be3-4ff9-89c9-735315fdc8f2

There is a JTextField that is supposed to show the name of the selected spritesheet. It is also supposed to let you change the spritesheet name with your input.
Neither of these behaviours work.

We might even consider dropping the JTextField entirely and letting users change the cell values in the sprite name list on the left.

**Comment by iamllcoolray** [comment]
Awesome! I'm going to look into this bug sometime next week. (My schedule just got busy)

**Comment by nightm4re94** [comment]
I suppose it's not going anywhere until then no hurry here 🙂

**Comment by iamllcoolray** [comment]
Thank you for being patient with me. I just finished my courses for this Fall semester and have more free time now. 
(It was brutal 😂)

So, I was trying to build Gradle in IntelliJ and got this error:

```
Build file 'C:\litiengine\build.gradle' line: 1

An exception occurred applying plugin request [id: 'de.gurkenlabs.semverguru', version: '1.3']
    > Failed to apply plugin 'de.gurkenlabs.semverguru'.
        > No match found
```

What do you think I should do to fix this?

**Comment by nightm4re94** [comment]
I believe this is #796 again. Have you made sure to compile with Java 21?

**Comment by iamllcoolray** [comment]
I just made the changes to Java 21 but I am still getting the same errors.

## Screenshots of changes:

![image](https://github.com/gurkenlabs/litiengine/assets/20143111/322dbdc3-62ac-493b-9668-99ed62f71102)

![image](https://github.com/gurkenlabs/litiengine/assets/20143111/707df867-0564-401a-aa32-09fdc9e5d9b9)

## Screenshot of error message:

`
Build file 'C:\litiengine\build.gradle' line: 1`

![image](https://github.com/gurkenlabs/litiengine/assets/20143111/69fffedc-6d41-46c5-b820-ecd8e5a6261a)

**Comment by iamllcoolray** [comment]
So I found a solution for the bug.


https://github.com/gurkenlabs/litiengine/assets/20143111/7998b084-388e-4567-88df-f45a2143ba80

I got the Sprite name in the text field which updates the Sprite's name in the application.

**Comment by iamllcoolray** [comment]
Saw this was still issue open but was resolved should be closed. Wasn't it supposed to close after the pull request was merged? Or did I not reference it properly?

**Comment by Gamebuster19901** [comment]
> Saw this was still issue open but was resolved should be closed. Wasn't it supposed to close after the pull request was merged? Or did I not reference it properly?

If you add the text

```Fixes #778``` or ```Resolves #778``` in a commit's message, it will close the issue when that commit is merged.

You can also put it anywhere in the first post of a pull request and it will also close the issue if the pull request is merged.

**Comment by iamllcoolray** [comment]
> > Saw this was still issue open but was resolved should be closed. Wasn't it supposed to close after the pull request was merged? Or did I not reference it properly?
> 
> If you add the text
> 
> `Fixes #778` or `Resolves #778` in a commit's message, it will close the issue when that commit is merged.
> 
> You can also put it anywhere in the first post of a pull request and it will also close the issue if the pull request is merged.
> 
> In the meantime though, we will have to wait for this to be closed manually by gurkenlabs.

Thank you so much. I just added ```Resolves #778``` to the pull request.

**Comment by Gamebuster19901** [comment]
> > > Saw this was still issue open but was resolved should be closed. Wasn't it supposed to close after the pull request was merged? Or did I not reference it properly?
> > 
> > 
> > If you add the text
> > `Fixes #778` or `Resolves #778` in a commit's message, it will close the issue when that commit is merged.
> > You can also put it anywhere in the first post of a pull request and it will also close the issue if the pull request is merged.
> > In the meantime though, we will have to wait for this to be closed manually by gurkenlabs.
> 
> Thank you so much. I just added `Resolves #778` to the pull request.

It won't work now because it's already been merged.
